src/game/default/g_util.c

```diff
@@ -547,6 +551,24 @@ g_team_t *G_OtherTeam(g_team_t *t) {
 }
 
 /*
+ *	Return the number of players on the given team.
+ */
+size_t G_TeamSize(g_team_t *team) {
+	g_client_t *cl;
+	size_t count = 0;
+	
+	for (int32_t i = 0; i < sv_max_clients->integer; i++){
+		if (!g_game.entities[i + 1].in_use)
+			continue;
+			
+		cl = g_game.entities[i + 1].client;
```

Comment:
Nitpick, but you could move `g_client_t *cl` into this scope, and declare as `const` too. C99, baby.